src/tests/functional/plugin/shared/include/snippets/mha.hpp

```diff
@@ -0,0 +1,47 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "shared_test_classes/base/snippets_test_utils.hpp"
+
+namespace ov {
+namespace test {
+namespace snippets {
+
+typedef std::tuple<
+        std::vector<ov::Shape>,          // Input shapes
```

Comment:
Minor: Maybe better to use `std::vector<ov::PartialShape>` everywhere, like in src/tests/functional/plugin/shared/include/snippets/matmul.hpp